style.css

```diff
@@ -4146,16 +4158,39 @@ ul {
   margin-bottom: 0;
 }
 
-.search-result-title {
-  font-size: 16px;
-  display: inline-block;
+.search-results .meta-group {
+  display: flex;
+  align-items: center;
+  clear: right;
```

Comment:
Is this `clear` rule needed?
As far as I know it should only work with floating elements.